build_number

```diff
@@ -0,0 +1 @@
+1308
```

Comment:
Are you adding these numbers or is this a new GitHub thing?

Reply:
Its kind of both - you can configure git to create this automatically. I am looking for a way to get a version string to be shown at the html page, but for now it is kind of half baken. This got accidentally to this pull request - i disabled the feature again and removed the file - ignore it for now. Maybe I come back to you for this feature at a later point in time...